dpti/workflows/examples/Water_Ice_Phase_Diagram/wget_graph.pb.sh

```
@@ -0,0 +1 @@
+wget https://huggingface.co/Felix5572/Water-Ice-SCAN-Phase-Dirgram/resolve/main/graph.pb
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)